test/unit/interface/OpMapIRTests.cpp

```diff
@@ -252,3 +253,43 @@ TEST_F(OpMapIRTestFixture, DialectOpOverloadTests) {
   EXPECT_EQ(map.lookup(Op1), "DialectOp4");
   EXPECT_EQ(map.lookup(Op2), "DialectOp4");
 }
+
+TEST_F(OpMapIRTestFixture, CallInstCoreOpMatchesInstructionTest) {
+  // Declare %OpaqueTy = type opaque
+  StructType *OpaqueTy = StructType::create(Context, "OpaqueTy");
+
+  // Define types
+  PointerType *OpaquePtrTy = PointerType::get(OpaqueTy, 0);
```

Comment:
Note that pointer types are opaque, so this is just a pointer. With a builder, it would be `Builder.getPtrTy()`.